Jenkinsfile

```diff
@@ -15,6 +15,12 @@ node('rhel8'){
 		sh "curl -Ls https://sh.jbang.dev | bash -s - app setup"
 		env.PATH="~/.jbang/bin:${env.PATH}"
 		sh "jbang trust add https://github.com/apache/"
+		//install cyclonedc-npm
+		sh "npm install --global @cyclonedx/cyclonedx-npm"
+		sh "cyclonedx-npm --omit dev --output-file node-sbom.json"

```

Comment:
thi cannot be called yet, the project was not built

Reply:
It is probably missing the dev dependencies. Running npm install should fix the problem. I added it in the "Install requirements" stage (although I did not see that step in the github actions).

Also where/when is the "camel-ls-sbom.json" file generated?

Reply:
the camel-ls-sbom file is generated when Camel Language Server is build. it is pushed as a Maven artifact. This project is downloading it during postinstall script execution https://github.com/camel-tooling/camel-lsp-client-vscode/blob/main/scripts/postinstall.js#L16